tekton/ci/shared/go_coverage_task.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: go-coverage
+  annotations:
+    description: |
+      Clones a repository from github into an empty-dir workspace.
```

Comment:
this description needs to be updated - add details about go coverage

Reply:
Thanks! I'm updating it.